run_jobs.py

```diff
@@ -118,15 +119,26 @@ def run_jobs(self) -> None:
 
 
 if __name__ == '__main__':
+    db_creator_obj = DBCreator(ENV['INVENTORY_DB'], ENV['APPEND_TABLE_NAMES'])
     how_started = os.environ.get('HOW_STARTED', None)
 
     if how_started == 'DOCKER_COMPOSE':
-        logger.info('Waiting for the MySQL turtle, hehe')
         # Wait for MySQL container to finish setting up
-        time.sleep(30.0)
+        logger.info('Waiting for the MySQL turtle')
+        waiting = True
+        while waiting:
```

Comment:
Rather than test for a Boolean `waiting` flag as the `while` loop condition,  check whether `db_creator_obj.conn is not None`.

Also, I think there should be a limit to the number of times it tries.